src/main/resources/assets/js/application/recruiter/services/tdprRecruiterSlotsService.js

```diff
@@ -0,0 +1,30 @@
+define(['angular', 'application/recruiter/tdprRecruiterModule', 'application/recruiter/services/tdprDateService', 'notification'], function (angular, tdprRecruiterModule) {
+    tdprRecruiterModule.service('tdprRecruiterSlotsService', ['$http', '$q', 'tdprDateService', 'dateFilter', 'AvailabilityEnum', function ($http, $q, tdprDateService, dateFilter, AvailabilityEnum) {
+
+        this.updateSlots = function (slots, personId, startDate, endDate) {
+            return $http.put("/api/slots/update/" + personId + "/" + dateFilter(startDate, "dd-MM-yyyy") + "/" + dateFilter(endDate, "dd-MM-yyyy"), this.reformatSlots(slots, personId)).then(function (response) {
```

Comment:
We need to change this endpoint to /api/slots  - without additional verbs - they are defined by the method. And person id should be a query param here.